### DISCUSSION TITLE
How does Jellyfin-web determine what content a client is capable of playing?

### DISCUSSION THREAD

**Discussion by kimboslice99** [jellyfin/jellyfin-web#5137] [Q&A] [answered]
It seems its using some method other than isTypeSupported, canPlayType or mediaCapabilities

I'm running Thorium with hardware decoding disabled so its using software decoding, h265 should play just fine, and does in Emby, but not Jellyfin

**Comment by thornbill** [comment, marked as answer]
It mainly uses `canPlayType` with some specific overrides for cases where support isn't reported correctly. 

https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/browserDeviceProfile.js

**Reply by kimboslice99** [reply]
Thing is, it doesnt seem to respect canPlayType, canPlayType reports probably for hevc, yet jellyfin wont allow directplay

running Thorium with software decoding, this works perfectly fine with Emby or any other app that can stream hevc, Jellyfin is the *only* one that cannot
